doc/config_rsmeval.rst

```diff
@@ -35,6 +35,12 @@ description *(Optional)*
 ~~~~~~~~~~~~~~~~~~~~~~~~
 A brief description of the experiment. This will be included in the report. The description can contain spaces and punctuation. It's blank by default.
 
+file_format *(Optional)*
+"""""""""""""""""""""""""""
+The format of the output files. Options are ``csv``, ``tsv``, or ``xlsx``. Defaults to ``csv`` if this is not specified.
```

Comment:
I'd suggest making this "intermediary files" to be consistent and making it a link to the relevant documentation page.